tests/src/functional/Drush/Traits/SiteUriTraitTest.php

```diff
@@ -51,6 +51,40 @@ public function testGetSitesSubdirFromUri(): void {
     $uri = $this->getSitesSubdirFromUri($drupal_root, "http://www.acquia.com");
     // If URI does't exist on sites.php file, fallback to site default.
     $this->assertEquals($uri, "default");
+
+    // If uri makes pattern port.domain.path and it exists in the sites.php
+    // then it return the value.
+    $uri = $this->getSitesSubdirFromUri($drupal_root, "http://acquia.com/stage/acquia_cms");
+    $this->assertEquals($uri, "stage.acquia_cms");
```

Comment:
This should be acquia.com instead stage.acquia_cms, right?

Reply:
If sites.php defined with the subdirectory then it return as it is otherwise format would port.domain.path i.e acquia.com.stage.acquia_cms would be return and try to search this format within the sites.php if it finds the same then subdirectory would return otherwise default but in above assertion stage.acquia_cms is defined in the sites.php for --uri=http://acquia.com/stage/acquia_cms